### PR TITLE
doc: reorganize known issues

### PR DESCRIPTION
**Pull request by greg-fer**
Edited structure of known issues to mirror code and doc structure. This should also get rid of the "catch-them-all" nRF5 section. NCSDK-19202.

### CODE REVIEW

**Review by peknis — APPROVED** [review]
I love it. As we are going to get rid of the old 0.3.0 and 0.4.0 releases anyway, I cannot see a reason why we could not leave out those tags in the known issues already now.

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -2102,113 +2325,29 @@ FW upgrade is broken for multi-image builds
...
-CIA-248: Samples with default SPM config fails to build for ``thingy91_nrf9160_ns``
-   All samples using the default SPM config fails to build for the ``thingy91_nrf9160_ns``  build target if the sample is not set up with MCUboot.
+NCSDK-10196: DFU fails for some configurations with the quick session resume feature enabled
+  Enabling :kconfig:option:`CONFIG_PELION_QUICK_SESSION_RESUME` together with the OpenThread network backend leads to the quick session resume failure during the DFU packet exchange.
+  This is valid for the :ref:`nRF52840 DK <ug_nrf52>` and the :ref:`nRF5340 DK <ug_nrf5340>`.
 
-   **Workaround:** Use the main branch.
+  **Workaround:** Use the quick session resume feature only for configurations with the cellular network backend.
 
 MCUboot
```
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:2347]: Should we move this to the end, next to Zephyr, TF-M etc.?

**Review by shanthanordic — CHANGES_REQUESTED** [review]
Some more alignment needed

1. 52 series stated as 53 series and 833 header under 52 series
2. thingy91 missing in91 series
3. When specifying HW issues, would it make sense to link PAN / errata in info center ?

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/known_issues.rst:2347]: Let's see how it's gonna look.

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -3101,37 +3254,10 @@ NCSDK-14015: Execution halts during boot
...
-.. rst-class:: v2-3-0 v2-2-0
-
-NCSDK-20095: Build warning in the RF test samples when the minimal pinout generic/Skyworks FEM is used
-  The :ref:`radio_test` and :ref:`direct_test_mode` samples build with a warning about generic/Skyworks FEM in minimal pinout configuration.
-
 Zephyr
-******
+======
+
+The issues in this section are related to the Zephyr downstream in the |NCS|.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply doc/nrf/known_issues.rst:3260]: Let's change the hierarchy there and see how it looks.

**Comment by greg-fer** [comment]
@jhn-nordic , @nordic-hani , we're changing the structure of known issues to better highlight issues that are related to hardware vs issues that affect NCS software. 

Please have a look at https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-10800/nrf/known_issues.html and comment here which known issues should be moved where to make this separation happen. For example, should issues like NCSDK-18746 and NCSDK-20026 come under libraries and samples respectively? Also, NCSDK-10106 might fit under Modem library?

**Comment by greg-fer** [comment]
Need to go back to drawing board with this one. Closing for now.

**Pull request closed without merging**